src/librustdoc/html/format.rs

```diff
@@ -1464,9 +1448,22 @@ impl clean::FnDecl {
             Some(n) => write!(f, "\n{})", Indent(n))?,
         };
 
-        fmt::Display::fmt(&self.output.print(cx), f)?;
+        fmt::Display::fmt(&self.print_output(cx), f)?;
         Ok(())
     }
+
+    pub(crate) fn print_output<'a, 'tcx: 'a>(
```

Comment:
```suggestion
    fn print_output<'a, 'tcx: 'a>(
```

unless I'm being blind this isn't being used anywhere else?

Reply:
Good catch! Wanna send a fix? :)